core/src/main/java/org/apache/accumulo/core/file/blockfile/cache/lru/LruBlockCache.java

```diff
@@ -520,7 +521,7 @@ public long getEvictedCount() {
 
     public EvictionThread(LruBlockCache cache) {
       super("LruBlockCache.EvictionThread");
-      setDaemon(true);
+      Threads.applyStandardsToThread(this, "LruBlockCache.EvictionThread");
```

Comment:
Maybe this should be turned into a Runnable instead so we use it with our regular utilities? It might not be necessary to create thread objects directly.

Or, if extending Thread is necessary, would it be cleaner to have a base Thread class to extend?